src/openms/source/IONMOBILITY/IMDataConverter.cpp

```diff
@@ -78,7 +78,7 @@ namespace OpenMS
     // fill up the PeakMaps by moving spectra from the input PeakMap
     for (const MSSpectrum& it : exp)
     {
-      split_peakmap[cv2index[it.getDriftTime()]].addSpectrum(std::move(it));
+      split_peakmap[cv2index[it.getDriftTime()]].addSpectrum(it);
```

Comment:
can't move const

Reply:
@timosachsenberg this is clearly a bug as the intent was to move the spectra into the `split_peakmap`. Is there any sane way to check that something has been "moved from"? I am thinking here 

https://github.com/OpenMS/OpenMS/blob/3c4a078ad4687677393c138d4acb91bb444a7c7b/src/tests/class_tests/openms/source/IMDataConverter_test.cpp#L74

for example, a `MSSpectrum` and `MSExperiment` should always be empty after it has been moved-from, so we could check for that in the tests?

Also `addSpectrum(&&)` is missing

Reply:
addressed in https://github.com/OpenMS/OpenMS/pull/5685